cmake/TileDB-Superbuild.cmake

```diff
@@ -48,6 +48,7 @@ set(INHERITED_CMAKE_ARGS
 if (TILEDB_TESTS)
   list(APPEND INHERITED_CMAKE_ARGS
     -DTILEDB_TESTS_AWS_S3_CONFIG=${TILEDB_TESTS_AWS_S3_CONFIG}
+    -DTILEDB_TESTS_AWS_S3_SKIP_INIT=${TILEDB_TESTS_AWS_S3_SKIP_INIT}
```

Comment:
Unless someone has specifically requested this, I think the the runtime configuration is sufficient. Let's remove this.